Readme.MD

```diff
@@ -102,7 +102,8 @@ Here is the list of parameters that must be maintained in the ```config.properti
   - ```OKTA_USERNAME``` is the username to use. If present will skip username input.
   - ```OKTA_PASSWORD``` is the password to use. If present will skip password input.
   - ```OKTA_BROWSER_AUTH``` set to **true** to use integrated web browser for authentication (default: **false**)
-  - ```OKTA_PROFILE``` is the name of the AWS profile to create/reuse (default: get AWS profile name based on per-session STS user name)  
+  - ```OKTA_PROFILE``` is the name of the AWS profile to create/reuse.  May also be specified on the commandline by ```--profile```. (default: get AWS profile name based on per-session STS user name)  
+  - ```OKTA_AWS_REGION``` is the default AWS region to store with the created profile.
```

Comment:
👍  I like this. You've clarified how your new command-line switch interacts with the existing settings and explained your new environment+config variable clearly.